BasicsCodelab/app/src/main/java/com/codelab/basics/MainActivity.kt

```diff
@@ -76,9 +77,28 @@ fun MyScreenContent(names: List<String> = listOf("Android", "there")) {
     }
 }
 
+@Composable
+fun NameList(modifier: Modifier, names: List<String>) {
```

Comment:
Have names as the first parameter and then modifier with a default value? In that case you don't have to name params when calling it

```suggestion
fun NameList(names: List<String>, modifier: Modifier = Modifier) {
```